docs/upgrading-to-v4.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading to v4
+
+This document lists the full set of breaking changes that need to be addressed when upgrading from version 3.x to 4.x, across both the "standard" and "attribution" builds (see [build options](/#build-options) for details).
+
+For a list of all changes see the [CHANGELOG](/CHANGELOG.md).
+
+## Standard build
+
+### General
+
+- The "base+polyfill" build has been removed (see [#435](https://github.com/GoogleChrome/web-vitals/pull/435))
+  - This includes the FID polyfill and the Navigation Timing polyfill supporting legacy Safari browsers.
+- Previously deprecated `getXXX()` functions have been removed (see [#435](https://github.com/GoogleChrome/web-vitals/pull/435))
+
+### `INPMetric`
+
+- `entries`
+  - Updated to only include entries with matching `interactionId` and `duration` values (previously it was only matching `interactionId` values, see [#442](https://github.com/GoogleChrome/web-vitals/pull/442)).
+
+## Attribution build
+
+### `INPAttribution`
```

Comment:
In the CHANGELOG you also mention "Add INP breakdown timings and LoAF attribution" is a breaking change. Should that be documented here as well?

Reply:
My intention there was to make it clear that that PR included breaking changes (the renaming/removing of attribution properties); however, the addition of the breakdown timings and LoAF attribution itself is not a breaking change (which I can see how that's confusing).

I wanted to keep the "Upgrading to v4" doc scoped as narrowly as possible to just the changes someone would need to make if they wanted to upgrade to the new version without necessarily adding any new features. Do you think it'd be better to include the new features as well?

Reply:
Either way SGTM. I guess it does seem useful for this doc to cover both breaking changes and new features for anyone interested in upgrading. Otherwise it could be easy to miss that this only covers breaking changes. Maybe different sections might make it clearer? Something like this:

## ⚠️ Breaking changes

...

## 🚀 New features

...

Reply:
Added.